istio.VERSION

```diff
@@ -1,9 +1,9 @@
 # DO NOT EDIT THIS FILE MANUALLY instead use
 # install/updateVersion.sh (see install/README.md)
-export CA_HUB="docker.io/istio"
-export CA_TAG="0.2.1"
-export MIXER_HUB="docker.io/istio"
-export MIXER_TAG="0.2.1"
-export ISTIOCTL_URL="https://storage.googleapis.com/istio-release/releases/0.2.1/istioctl"
-export PILOT_HUB="docker.io/istio"
-export PILOT_TAG="0.2.1"
+export CA_HUB="gcr.io/istio-testing"
+export CA_TAG="678763110b0d49f0a1eb417c7313616f67cce1f0"
+export MIXER_HUB="gcr.io/istio-testing"
+export MIXER_TAG="a37507927b55d1e901120de20fc0fa3fbf0f1926"
+export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/de48f4f7ca5ad7d29b52e882b61d4386e4645dd9/artifacts/istioctl"
+export PILOT_HUB="gcr.io/istio-testing"
+export PILOT_TAG="de48f4f7ca5ad7d29b52e882b61d4386e4645dd9"
```

Comment:
this is 4 days old @chxchx !
probably because the post submit is broken?

Reply:
I agree. The Latest SHAs we use for pilot, mixer, auth are read off from the head of stable branch, which is fast forward to a just merged PR if it passes postsubmit. I just checked prow and postsubmit has been green today, so the next auto generated PR to update dependency will pick them up.

Reply:
it took a lot effort to get the presubmit pr in yesterday but we made it happen

Reply:
legend!